clang/lib/Basic/Targets/AArch64.cpp

```diff
@@ -399,7 +399,14 @@ void AArch64TargetInfo::getTargetDefines(const LangOptions &Opts,
   Builder.defineMacro("__AARCH64_CMODEL_" + CodeModel + "__");
 
   // ACLE predefines. Many can only have one possible value on v8 AArch64.
-  Builder.defineMacro("__ARM_ACLE", "200");
+  Builder.defineMacro("__ARM_ACLE_VERSION(year, quarter, patch)",
+                      "100 * (year) + 10 * (quarter) + (patch)");
```

Comment:
I'd add a surrounding parentheses to avoid precedence issues with the macro. 
```suggestion
                      "(100 * (year) + 10 * (quarter) + (patch))");
```